src/transformers/models/gpt2/modeling_gpt2.py

```diff
@@ -369,9 +369,9 @@ class GPT2DoubleHeadsModelOutput(ModelOutput):
             Prediction scores of the language modeling head (scores for each vocabulary token before SoftMax).
         mc_logits (:obj:`torch.FloatTensor` of shape :obj:`(batch_size, num_choices)`):
             Prediction scores of the multiple choice classification head (scores for each choice before SoftMax).
-        past_key_values (:obj:`List[torch.FloatTensor]`, `optional`, returned when ``use_cache=True`` is passed or when ``config.use_cache=True``):
-            List of :obj:`torch.FloatTensor` of length :obj:`config.n_layers`, with each tensor of shape :obj:`(2,
-            batch_size, num_heads, sequence_length, embed_size_per_head)`).
+        past_key_values (:obj:`Tuple[Tuple[torch.Tensor]]`, `optional`, returned when ``use_cache=True`` is passed or when ``config.use_cache=True``):
+            :obj:`Tuple[Tuple[torch.Tensor]]` of length :obj:`config.n_layers`, with each tuple having 2 tuples each of
+            which has a tensor of shape :obj:`(batch_size, num_heads, sequence_length, embed_size_per_head)`).
```

Comment:
Let's use plain English in the description, the type is already written above.
```suggestion
           Tuple of length :obj:`config.n_layers`, containing tuples of tensors of shape
            :obj:`(batch_size, num_heads, sequence_length, embed_size_per_head)`).
```